### ISSUE TITLE
Static Variables Not Static and Possible Memory Leak?

### ISSUE THREAD

**Issue by jpbro** [WaynePhillipsEA/twinbasic#618]
**Describe the bug**
It appears that tB leaks memory when using Static variables. The value stored in the static variable doesn't hold between method calls, and every call to the method holding the Static variable appears to leak memory.

**To Reproduce**
1. Start a new tB EXE project.
2. Add the following code:

```
Public Sub Main
   Do
      GetStatic
   Loop
End Sub

Private Sub GetStatic() As String
   Static MyVar As String

   If MyVar = vbNullString Then
      MyVar = "ABC"
      Debug.Print MyVar   ' Would expect to see this printed once, but it prints hundreds of thousands of times
   End If
   GetStatic = MyVar
End Sub
```
3. Run Sub Main

**Expected behavior**
I would expect Sub Main to run infinitely, but only print "ABC" to the Debug Console once.

Instead "ABC" gets printed to the console hundreds of thousands of times and the memory used by twinBASIC_win32.exe climbs continuously.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.13.59

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting!  Looks like it is a String-specific bug, as changing to the static variable to a Variant appears to correct the problem.  I will look into this in more detail first thing next week and get it fixed.

**Comment by WaynePhillipsEA** [comment]
Ah, I've spotted the bug.  It's the return-value optimization that was introduced in v0.13.19.   It shouldn't be allowing that optimization when the return value is from a static variable.  I should be able to get that fixed for you in the morning.

**Comment by jpbro** [comment]
That's awesome @WaynePhillipsEA, thank you very much and have a Happy New Year!

**Comment by WaynePhillipsEA** [comment]
This is now fixed in v0.13.60, thanks for reporting!  

> That's awesome @WaynePhillipsEA, thank you very much and have a Happy New Year!

Thanks and you have a Happy New Year too @jpbro!
